cloudmarker/stores/mongodbstore.py

```diff
@@ -0,0 +1,100 @@
+"""MongoDB store plugin."""
+
+
+import logging
+
+from pymongo import MongoClient
+
+_log = logging.getLogger(__name__)
+
+
+class MongoDBStore:
+    """A plugin to store records on the MongoDB."""
+
+    def __init__(self, host, port, username=None, password=None, db='gcp',
```

Comment:
Depending on the [comment](https://github.com/cloudmarker/cloudmarker/pull/20/commits/9efa467d41b3f51701f5bb509ea3adfa609d3cf1#r250082197), the args list should be something like:
`def __init__(self, username, password, host, port=27017, db='gcp', buffer_size=1000)`
This would allow more cohesiveness with the `__init__` and defaults used by mongo client.

Reply:
`db` is default argument, how to keep default argument before non-default.
I have used the default name for the `db` which is `gcp` so that in case if we plan to integrate this application(or its store that is MongoDB) to some dashboard then it will easily in application control and would require less configurations, if this field is configured by a user of the application (which is possible now as well) then we need some way of keeping track of that name for the dashboard as well.

Reply:
Okay, let me present this in a different way. Usually, default host and port for any database is `localhost` and standard port for the database type. And in all cases, you need to give the DB name. You can have all params as default to make it work OOB, but it's not very practical. It is a very common scenario that people are using shared databases, so having a DB name as a required arg does not make config a lot difficult but just a regular config.

Reply:
Created issue #28 to make a decision about this and make the required changes in a separate commit.